### PR TITLE
Fixed List with EndTags

### PR DESCRIPTION
**Pull request by AntonBoch1244**
This fix allows parsing empty lists.

### CODE REVIEW

**Comment by AntonBoch1244** [comment]
Waiting for review or merge.

**Comment by TkTech** [comment]
Do you have an example NBT file for this issue?

**Comment by AntonBoch1244** [comment]
@TkTech I upload real Gzipped NBTFile(Generated by server with MinecraftForge and FML installed.), not just example.
[ExampleforEmptylistIssue.zip](https://github.com/TkTech/PyNBT/files/1368580/ExampleforEmptylistIssue.zip)

**Comment by AntonBoch1244** [comment]
And upload fabricated NBTFile with zero length list.
[CompoundWithZeroLengthList.nbt.gz](https://github.com/TkTech/PyNBT/files/1368732/CompoundWithZeroLengthList.nbt.gz)

**Comment by AntonBoch1244** [comment]
And another fabricated NBTFile example.
[CompoundWithListOfTenEndTags.nbt.gz](https://github.com/TkTech/PyNBT/files/1368750/CompoundWithListOfTenEndTags.nbt.gz)

**Comment by AntonBoch1244** [comment]
Tested dumping of files.

Zero length list
```
TAG_Compound(u''): 1 entries
{
  TAG_List(u''): 0 entries
  {
  }
}
```

List with 10 end tags. (In NBTExplorer behavior are broken :D)
```
TAG_Compound(u'EndTagsListTest'): 1 entries
{
  TAG_List(u'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'): 0 entries
  {
  }
}
```

**Comment by AntonBoch1244** [comment]
Squashed.

File: pynbt/nbt.py [hunk 1]
```diff
@@ -196,6 +199,9 @@ def pretty(self, indent=0, indent_str='  '):
 class TAG_String(BaseTag):
     __slots__ = ('name', 'value')
 
+class TAG_End(BaseTag):
```
Comment (TkTech) [inline pynbt/nbt.py]: Missing newline (PEP8)

**Pull request merged**